src/handlers/multiSig/motionCanceled.ts

```diff
@@ -0,0 +1,47 @@
+import { mutate } from '~amplifyClient';
+import {
+  UpdateColonyMultiSigDocument,
+  UpdateColonyMultiSigInput,
+  UpdateColonyMultiSigMutationVariables,
+} from '~graphql';
+import { ContractEvent } from '~types';
+import { getMultiSigClient, verbose } from '~utils';
+import { getMultiSigDatabaseId } from './helpers';
+import { getChainId } from '~provider';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const {
+    args: { motionId },
+    colonyAddress,
+  } = event;
+
+  if (!colonyAddress) {
+    return;
+  }
+
+  const multiSigClient = await getMultiSigClient(colonyAddress);
+
+  if (!multiSigClient) {
+    return;
+  }
```

Comment:
I think you might be able to get the address from the event's `contractAddress` property without having to get the client.